rebar.lock

```diff
@@ -59,7 +59,7 @@
  {<<"hackney">>,{pkg,<<"hackney">>,<<"1.17.0">>},0},
  {<<"helium_proto">>,
   {git,"https://github.com/helium/proto.git",
-       {ref,"c10a73fc89d895f439bc82efbbc8cd11ed880546"}},
+       {ref,"f1208e25b8966e0b480023439a53f5e159f9c0d0"}},
```

Comment:
Why proto update?

Reply:
I rebased onto master from when I started the branch and needed to update proto, this was the version that came along, I'll revert.